invokeai/app/services/shared/invocation_context.py

```diff
@@ -194,13 +194,20 @@ def save(
             node_id=self._context_data.invocation.id,
         )
 
-    def get_pil(self, image_name: str) -> Image:
+    def get_pil(self, image_name: str, format: str | None = None) -> Image:
```

Comment:
In `invocations/images.py` we have a literal for this
```py
IMAGE_MODES = Literal["L", "RGB", "RGBA", "CMYK", "YCbCr", "LAB", "HSV", "I", "F"]
```

Can you please move this constant into `invocations/constants.py` and use that here instead of `str|none`?

Also, PIL calls this "mode" not format